src/popup/components/AccountInfo.vue

```diff
@@ -90,11 +95,19 @@ export default defineComponent({
     showProtocolIcon: Boolean,
     dense: Boolean,
     showExplorerLink: Boolean,
+    isChainName: Boolean,
```

Comment:
To be honest I don't fully understand this prop. The `AccountInfo` component should display the account info based on the `account` data. So it should establish if the AE chain name should be displayed on its own. Or I'm missing something?

Reply:
Do you suggest checking this `address.startsWith('nm_')` in the `AccountInfo`?